tests/vlan_test.py

```diff
@@ -626,8 +1155,8 @@ def test_config_set_router_port_on_member_interface(self):
         result = runner.invoke(config.config.commands["interface"].commands["ip"].commands["add"],
                                ["Ethernet4", "10.10.10.1/24"], obj=obj)        
         print(result.exit_code, result.output)
-        assert result.exit_code == 0
-        assert 'Interface Ethernet4 is a member of vlan' in result.output
+        assert result.exit_code != 0
```

Comment:
Why the exit_code is different with that without change? It seems introduce backward compatibility issue.